yarn.lock

```diff
@@ -5649,6 +5649,11 @@ fs.realpath@^1.0.0:
   resolved "https://registry.yarnpkg.com/fs.realpath/-/fs.realpath-1.0.0.tgz#1504ad2523158caa40db4a2787cb01411994ea4f"
   integrity sha1-FQStJSMVjKpA20onh8sBQRmU6k8=
 
+fs@~0.0.1-security:
+  version "0.0.1-security"
+  resolved "https://registry.yarnpkg.com/fs/-/fs-0.0.1-security.tgz#8a7bd37186b6dddf3813f23858b57ecaaf5e41d4"
+  integrity sha1-invTcYa23d84E/I4WLV+yq9eQdQ=
+
```

Comment:
(non-blocking:) Didn't you fix this in https://github.com/jupyterlab/jupyterlab/pull/6889 already?

Reply:
yep. The branch probably needs an integrity run

Reply:
Done! Although there's still one little bothersome difference in `yarn.lock` for no apparent reason. Annoying

Reply:
Annnd now that's taken care of too. Hooray!